packages/fcl/src/current-user/index.js

```diff
@@ -367,6 +426,18 @@ async function signUserMessage(msg) {
   }
 }
 
+/**
+ * @description - Current User
+ * 
+ * @returns {Object} - currentUser
+ * @returns {Function} currentUser.authenticate - Authenticate the current user
```

Comment:
Can you reference the types above here?

Reply:
@justinbarry They are different things. I'll make that more clear in the description though.